x/mint/simulation/genesis.go

```diff
@@ -35,7 +34,7 @@ func RandomizedGenState(simState *module.SimulationState) {
 		Staking:          sdk.NewDecWithPrec(5, 1), // 0.5
 		PoolIncentives:   sdk.NewDecWithPrec(3, 1), // 0.3
 		DeveloperRewards: sdk.NewDecWithPrec(2, 1), // 0.2
-	}, "", time.Time{})
+	}, "", 0)
```

Comment:
This being the only test I had to make a change to suggests that we need to test the start time delay more thoroughly

Reply:
EDIT: Nope, I just missed the relevant test in hooks_test